src/ds5/ds5-options.cpp

```diff
@@ -666,7 +666,7 @@ namespace librealsense
 
     void auto_exposure_limit_option::set(float value)
     {
-        if (!is_valid(value))
+        if (!is_valid(value) && value != 0)
```

Comment:
The control's range to be adjusted - zero should be included, unless the control doesn't start with 1.
Please check the actual range with D415 and D435 and update